3p/3p.js

```diff
@@ -232,7 +232,8 @@ export function validateData(data, mandatoryFields, opt_optionalFields) {
       allowedFields = allowedFields.concat(field);
     } else {
       userAssert(
-        data[field],
```

Comment:
Sounds like `false` value would be deemed invalid in the past. Strange that no one noticed.

Reply:
yeah its odd. `false` would fail but I think `"false"` was ok, so maybe everyone was using the string version 🤷‍♂️